pkgs/development/python-modules/robotframework/default.nix

```diff
@@ -10,6 +10,8 @@ buildPythonPackage rec {
     extension = "zip";
   };
 
+  propagatedBuildInputs = [ tkinter ];
+
```

Comment:
`pythonImportsCheck` probably would have caught this

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
I added in the unit tests, but there's a problem. See https://discourse.nixos.org/t/mktime-out-of-range-during-python-unit-tests/8848.